turbinia/workers/psort.py

```diff
@@ -42,13 +81,13 @@ def run(self, evidence, result):
     psort_evidence = PlasoCsvFile(source_path=psort_file)
     psort_log = os.path.join(self.output_dir, '{0:s}.log'.format(self.id))
 
-    cmd = ['psort.py', '--status_view', 'none', '--logfile', psort_log]
-    if config.DEBUG_TASKS or evidence.config.get('debug_tasks'):
+    cmd = self.build_plaso_command('psort.py', self.task_config)
+
+    cmd.extend(['--logfile', psort_log])
+    if config.DEBUG_TASKS or self.task_config['debug_tasks']:
       cmd.append('-d')
 
-    cmd.extend(['--additional_fields', 'yara_match'])
-    cmd.extend(['-w', psort_file, evidence.local_path])
-    cmd.extend(['--temporary_directory', self.tmp_dir])
```

Comment:
I think we still want this temporary directory line.  We want the output to go to a temp file since sometimes plaso has issues with the shared storage filesystems (e.g. NFS) .

Reply:
Should we make this part of the task_config?